### ISSUE TITLE
Quarto yaml validation fails for include-in-header: text: for PDFs

### ISSUE THREAD

**Issue by AnthonyRaborn** [quarto-dev/quarto-cli#897]
I am wanting to use Quarto to create a book-based website and a PDF document from the same code. Using RStudio, I created a new Quarto Book project with the default files, which works and produces both a website and PDF from the same documents. However, when I start trying to include custom LaTeX code through the _quarto.yml file, I get an error.

The specific _quarto.yml file I am using alongside the default files included with the quarto project through RStudio is as follows:

```
project:
  type: book

book:
  title: "quarto-test-project"
  author: "Jane Doe"
  date: "5/16/2022"
  chapters:
    - index.qmd
    - intro.qmd
    - summary.qmd
    - references.qmd

bibliography: references.bib

format:
  html:
    theme: cosmo
  pdf:
    include-in-header:
      text: |
        \usepackage{eplain}
        \usepackage{easy-todo}
editor: visual
```

The only change to this file versus the original file is the include-in-header lines, which I took from the Quarto example on the website. This produces the following error:

```
Project _quarto.yml validation failed.

In file C:\Users\location\to\the\quarto-test-project\_quarto.yml
(line 20, column 5 through line 23, column 31) Key pdf has value include-in-header:
      text: |
        \usepackage{eplain}
        \usepackage{easy-todo}, which must be 'default'
```

The expected outcome is that the project would run, producing the website and the PDF where the PDF has the designated text included in the header of the TeX file. I don't see anything in the documentation on the website that indicates that this is an invalid way to organize and use a Quarto project.

I am using:

-  quarto version 0.9.193
- RStudio 2022.02.1+461 "Prairie Trillium" Release (8aaa5d470dd82d615130dbf663ace5c7992d48e3, 2022-03-17) for Windows
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.8 Chrome/69.0.3497.128 Safari/537.36
- R version 4.1.3 (2022-03-10)
- Platform: x86_64-w64-mingw32/x64 (64-bit)
- Running under: Windows 10 x64 (build 19044)

**Comment by cscheid** [comment]
Hi - 

That is a very recent feature which wasn't available on 0.9.193. Can you try upgrading to the latest quarto (or latest RStudio daily, in your case https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/RStudio-2022.06.0-daily-406.exe)? Your example worked for me locally.

**Comment by AnthonyRaborn** [comment]
The upgrade to the newer version of quarto did it, thanks!
